src/components/CourseForm/styles.module.css

```diff
@@ -1,12 +1,25 @@
-form > div {
+.container {
   display: flex;
-  justify-content: space-between;
-  margin-bottom: 24px;
+  flex-direction: column;
 }
 
-textarea {
+.container textarea {
```

Comment:
Why do we need .`container` class here? I think `textarea` is enough (for `form` we din't have .`container`) and according to the jsx it should be `description`.

Reply:
fixed